test/src/Makefile

```diff
@@ -0,0 +1,50 @@
+PACKAGE  = terraform-aws-ecs-container-definition
+GOEXE   ?= /usr/bin/go
+GOPATH   = $(CURDIR)/.gopath
+GOBIN    = $(GOPATH)/bin
+BASE     = $(GOPATH)/src/$(PACKAGE)
+PATH 	:= $(PATH):$(GOBIN)
+
+export TF_DATA_DIR ?= $(CURDIR)/.terraform
+export TF_CLI_ARGS_init ?= -get-plugins=true
+export GOPATH
+
+.PHONY: all
+## Default target
+all: test
+
+ifneq (,$(wildcard /sbin/apk))
+## Install go, if not installed
+$(GOEXE):
+	apk add --update go
+endif
+
+ifeq ($(shell uname -s),Linux)
```

Comment:
@osterman considering the conversation [here](https://sweetops.slack.com/archives/G014YEKDH4K/p1593207364179000), maybe we update this to include `Darwin` as well since it works on OSX? 

```
ifeq ($(shell uname -s),$(filter $(shell uname -s),Linux Darwin))
```